packages/stacks/src/stacks.utils.ts

```diff
@@ -71,3 +73,110 @@ export function getStacksBurnAddress(chainId: ChainID): string {
       return 'ST000000000000000000002AMW42H';
   }
 }
+
+function cleanHex(hexWithMaybePrefix: string): string {
+  if (hexWithMaybePrefix !== 'string') return hexWithMaybePrefix;
+  return hexWithMaybePrefix.startsWith('0x')
+    ? hexWithMaybePrefix.replace('0x', '')
+    : hexWithMaybePrefix;
+}
+
+export function hexToBuff(hex: string): Buffer {
+  return Buffer.from(cleanHex(hex), 'hex');
+}
+
+export const stacksValue = ({
+  value,
+  fixedDecimals = true,
+  withTicker = true,
+  abbreviate = false,
+}: {
+  value: number | string | BigNumber;
+  fixedDecimals?: boolean;
+  withTicker?: boolean;
+  abbreviate?: boolean;
+}) => {
+  const stacks = microStxToStx(value);
+  const stxAmount = stacks.toNumber();
+  return `${
+    abbreviate && stxAmount > 10000
+      ? abbreviateNumber(stxAmount)
+      : stxAmount.toLocaleString('en-US', {
+          maximumFractionDigits: fixedDecimals ? STX_DECIMALS : 3,
+        })
+  }${withTicker ? ' STX' : ''}`;
+};
+
+/**
+ * Gets the contract name of a fully qualified name of an asset.
+ *
+ * @param contractId - the source string: [principal].[contract-name] or [principal].[contract-name]::[asset-name]
+ */
+export const getStacksContractName = (contractId: string): string => {
+  if (contractId.includes('.')) {
+    const parts = contractId?.split('.');
+    if (contractId.includes('::')) {
+      return parts[1].split('::')[0];
+    }
+    return parts[1];
+  }
+  // eslint-disable-next-line no-console
+  console.warn(
```

Comment:
I think the logs here need to go so they don't get published. Unless we will lock versions until all work is finished which is OK too